docs/my-website/package.json

```diff
@@ -22,6 +22,7 @@
     "prism-react-renderer": "^1.3.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "sharp": "^0.32.5",
```

Comment:
what does this do

Reply:
Hm, wasn't building without it 🤔 Can remove and try again